client/my-sites/plans/current-plan/current-plan-thank-you/thank-you.js

```diff
@@ -30,11 +32,15 @@ export class ThankYouCard extends Component {
 			showCalypsoIntro,
 			showContinueButton,
 			showHideMessage,
-			siteSlug,
 			title,
 			translate,
 		} = this.props;
 
+		const dismissUrl =
+			this.props.queryArgs && 'install' in this.props.queryArgs
+				? addQueryArgs( { install: this.props.queryArgs.install }, this.props.currentRoute )
+				: this.props.getCurrentRoute;
```

Comment:
Minor: Maybe destructure `queryArgs` and `currentRoute` from `this.props` in the variable decl above? Both are used at least twice here.

Reply:
I'm glad you mentioned that because this line is _wrong_!

```suggestion
				: this.props.currentRoute;
```

Reply:
Heh I missed that though :sweat_smile: